modules/mailer/mail.go

```diff
@@ -126,11 +125,16 @@ func SendIssueNotifyMail(u, owner *models.User, repo *models.Repository, issue *
 		return tos, nil
 	}
 
-	subject := fmt.Sprintf("[%s] %s (#%d)", repo.Name, issue.Name, issue.Index)
-	content := fmt.Sprintf("%s<br>-<br> <a href=\"%s%s/%s/issues/%d\">View it on Gogs</a>.",
-		markdown.RenderSpecialLink([]byte(strings.Replace(issue.Content, "\n", "<br>", -1)), owner.Name+"/"+repo.Name, repo.ComposeMetas()),
-		setting.AppUrl, owner.Name, repo.Name, issue.Index)
-	msg := NewMessage(tos, subject, content)
+	var (
+		subject  = fmt.Sprintf("[%s] %s (#%d)", repo.Name, issue.Name, issue.Index)
+		gogsLink = fmt.Sprintf("%s%s/%s/issues/%d", setting.AppUrl, owner.Name, repo.Name, issue.Index)
+		content  = fmt.Sprintf("%s<br>&mdash;<br><a href=\"%s\">View it on Gogs</a>.",
+			markdown.Render([]byte(issue.Content), repo.RepoLink(), repo.ComposeMetas()), gogsLink)
+		// Markdown is meant to look good even when it can't be parsed into HTML, that's why it's being
+		// written directly in the email for the text/plain body (fallback version).
+		contentPlain = fmt.Sprintf("%s\n\n---\nView it on gogs: %s", issue.Content, gogsLink)
+		msg          = NewMessage(tos, subject, content, contentPlain)
+	)
```

Comment:
Thanks for pointing out, the way code is done looks confusing, I recommend use `varName := xxx` instead.